packages/core/src/web/app/components/dialogs/camera/AdorCalibrationV2.tsx

```diff
@@ -196,31 +153,6 @@ const AdorCalibrationV2 = ({ factoryMode = false, onClose }: Props): React.JSX.E
     );
   }
 
-  if (step === Step.FIND_CORNER) {
-    return (
-      <FindCorner
-        onBack={() => setStep(Step.PUT_PAPER)}
-        onClose={onClose}
-        onNext={async () => {
-          progressCaller.openNonstopProgress({ id: PROGRESS_ID, message: lang.device.processing });
-
-          const levelingData = await getLevelingData('hexa_platform');
-          const refHeight = levelingData.E;
-
-          Object.keys(levelingData).forEach((key) => {
-            levelingData[key] = refHeight - levelingData[key];
-          });
-          updateParam({ levelingData });
-          await saveCheckPoint(calibratingParam.current);
-          progressCaller.popById(PROGRESS_ID);
-          setStep(Step.ELEVATED_CUT);
-        }}
-        updateParam={updateParam}
-        withPitch={withPitch}
-      />
-    );
-  }
-
```

Comment:
These steps were used to let user engrave chessboard points to calibrate. Not used anymore.